modyn/supervisor/internal/pipeline_executor/models.py

```diff
@@ -492,6 +492,7 @@ def materialize(self, log_dir_path: Path, mode: Literal["initial", "increment",
             pipeline_logdir.rename(backup_dir)
 
         pipeline_logdir.mkdir(parents=True, exist_ok=True)
+        (pipeline_logdir / ".name").write_text(self.config.pipeline.pipeline.name)
```

Comment:
Why this? We can infer this from the pipeline log, so why have another file?

Reply:
In the tool that I built for comparing/visualizing pipeline runs I don't want to load the whole file for only showing the user a list of available pipeline runs. Only after the user selection should it be loaded. To make the runs identifiable, I need some identifier that is easily accessible. We could alternatively also just rename the logfile that is currently named `pipeline.log`.

Reply:
Yeah, then let's rename the file to `<pipeline_name>.log` :)

Reply:
Will do in a followup PR, I don't wont to kick off the CI again here for that